Code/BuildSystem/AzurePipelines/Windows-x64.yml

```diff
@@ -90,9 +104,9 @@ jobs:
     inputs:
       cmakeArgs: -DCMAKE_PREFIX_PATH=$(System.DefaultWorkingDirectory)\ThirdPartyWinx64\Qtvs141x64; -DEZ_QT_DIR="$(System.DefaultWorkingDirectory)\ThirdPartyWinx64\Qtvs141x64" -DEZ_ENABLE_FOLDER_UNITY_FILES=0 -DEZ_BUILD_OPENXR=1 -DEZ_BUILD_PHYSX=1 -DEZ_BUILD_FMOD=1 -DEZ_BUILD_GAMES=1 -DEZ_COMPILE_ENGINE_AS_DLL=1 -DEZ_ENABLE_QT_SUPPORT=1 -DCMAKE_SYSTEM_VERSION=10.0.18362.0 -DCMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10.0.18362.0 -A "x64" -G "Visual Studio 16 2019" ../
   - task: MSBuild@1
-    displayName: Build solution build/ezVs2019.sln
+    displayName: Build solution build/s_build.sln
```

Comment:
Where does this name "s_build" come from? I don't see that we are setting the solution name anywhere.

Reply:
This name is being auto-generated this way. Normally the root folder of the checkout is called ezEngine so the sln is called ezEngine_build.sln. However, on CI the checkout folder is callsed 's'. So s_build.sln it is.
This is cached so only found this out once I cleaned the build.

Reply:
Use -DEZ_SOLUTION_NAME:string="solution" that will hardcode the solution name and you don't need to wonder what the folder name is. It's what I used in other github actions.

Reply:
You PR actually removed the cache on CI and made my PR fail to build as well, so I had to make this change there as well. You'll probably get a conflict because of this.